code/game/objects/structures/false_walls.dm

```diff
@@ -110,6 +110,9 @@
 		else
 			to_chat(user, "<span class='warning'>You can't reach, close it first!</span>")
 
+	else if(istype(src, /obj/structure/falsewall/reinforced))
+		if(istype(W, /obj/item/wirecutters))
+			dismantle(user, TRUE)
```

Comment:
Dismantling a reinforced falsewall now sounds like welding it, but I didn't figure out how to fix that with "playsound(src.loc, W.usesound, 100, 1)".

Reply:
pass the tool through so you can play the right sound in the dismantle proc maybe

Reply:
I tried what you said, didn't work. Simplified it and still didn't work. There's no sound at all and the false wall doesn't even deconstruct, it's weird. Here's my code:

/obj/structure/falsewall/proc/dismantle(mob/user, disassembled = TRUE, obj/item/tool)
	user.visible_message("[user] dismantles the false wall.", "<span class='notice'>You dismantle the false wall.</span>")
	playsound(loc, tool.usesound, 100, 1)
	deconstruct(disassembled)

Reply:
you can't do that, you can't have a optional keyword arg like disassembled=TRUE followed by a non optional one like obj/item/tool

either refactor it to be
`dismantle(mob/user, obj/item/tool, disassembled=TRUE)`

and update every single call to dismantled to be true, or do what I told you to do
```
dismantle(mob/user, disassembled=TRUE, obj/item/tool = null)
    ...
    if(tool)
      playsound(loc, tool.usesound, 100, 1)
   else
     playsound(loc, 'welder.ogg', 100, 1)
    ...
```

Reply:
Ok thanks, I did the latter, but I debugged it and it doesn't recognize the wirecutter as a tool. The reinforced falsewall can be wirecut but it sounds like a welder. Code:
```
/obj/structure/falsewall/proc/dismantle(mob/user, disassembled=TRUE, obj/item/tool = null)
	user.visible_message("[user] dismantles the false wall.", "<span class='notice'>You dismantle the false wall.</span>")
	if(tool)
		playsound(src, tool.usesound, 100, 1)
	else
		playsound(src, 'welder.ogg', 100, 1)
	deconstruct(disassembled)
```

Reply:
did you update the call from the false wall to pass in the wirecutteR?

Reply:
Done, works now.

Reply:
nice work!